jwql/tests/test_setup_info.py

```diff
@@ -0,0 +1,28 @@
+#! /usr/bin/env python
+"""Tests for the ``setup.py`` module.
```

Comment:
@bhilbert4 To be consistent with the names of the other tests, I would suggest renaming this module to `test_setup.py`

Reply:
Renamed.